### PR TITLE
Edit ko_kr.json

### PR DESCRIPTION
**Pull request by UnineVesiKass**
Add Korean Translation

### CODE REVIEW

File: Xplat/src/main/resources/assets/botania/lang/ko_kr.json [hunk 1]
```diff
@@ -1480,12 +1480,12 @@
   "block.minecraft.banner.botania.flower.blue": "신비로운 파란색 꽃",
   "block.minecraft.banner.botania.flower.purple": "신비로운 보라색 꽃",
   "block.minecraft.banner.botania.flower.cyan": "신비로운 청록색 꽃",
-  "block.minecraft.banner.botania.flower.light_gray": "신비로운 밝은 회색 꽃",
+  "block.minecraft.banner.botania.flower.silver": "신비로운 밝은 회색 꽃",
```
Comment (TheRealWormbo) [inline Xplat/src/main/resources/assets/botania/lang/ko_kr.json]: Please revert the change to light gray and light blue translation key names for banner patterns, those will not work when they are called "silver" and "lightBlue", respectively.

**Comment by UnineVesiKass** [comment]
I fix it!
